alacritty_terminal/src/term/mod.rs

```diff
@@ -767,7 +686,7 @@ pub struct Term<T> {
     tabs: TabStops,
 
     /// Mode flags.
-    mode: TermMode,
+    pub mode: TermMode,
```

Comment:
In general, I don't like that change, the thing is that mode is something
that set/unset by various side effects, not by the user directly and the user
can only read this field, not write to it various stuff. I do understand that
you wanted to remove a bunch of brackets, but having API that prevents you from
stupid things like unsetting mode yourself on the fly was kind of making sense.